perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -62,6 +62,7 @@
 MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```

Comment:
This comment applies to the overall PR:

it contains a number of things, plot per second graph is just one of them. The other one is it has a driver script (.sh) fie. Would you consider splitting them into two? Or if you prefer to get these in as one PR, please clarify in the PR description about this.